### PR TITLE
mobskills scripts

### PR DESCRIPTION
**Pull request by eraffxi**

### CODE REVIEW

**Comment by xdemolish** [comment]
o god "/scripts" hath returned

o, and amend commit and change the message to 
SCRIPTS: added mobskills for x,y,z
or w/e

**Comment by eraffxi** [comment]
corrected

File: scripts/globals/mobskills/Call_of_the_moon1.lua [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+---------------------------------------------------
+
+require("scripts/globals/settings");
+require("scripts/globals/status");
+require("scripts/globals/monstertpmoves");
+
+---------------------------------------------------
+
+function OnMobSkillCheck(target,mob,skill)
+ if (VanadielHour() >= 19 or VanadielHour() <= 6) then
```
Comment (TeoTwawki) [inline scripts/globals/mobskills/Call_of_the_moon1.lua]: WHERE MAH INDENT?!

:grin:

File: scripts/globals/mobskills/Call_of_the_moon2.lua [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+require("scripts/globals/status");
+require("scripts/globals/monstertpmoves");
+
+---------------------------------------------------
+
+function OnMobSkillCheck(target,mob,skill)
+ if (VanadielHour() >= 7 and VanadielHour() <= 18) then 
+ --  printf("Good to go Moon2");
+	return 0;
+	else
```
Comment (TeoTwawki) [inline scripts/globals/mobskills/Call_of_the_moon2.lua]:
```lua
function OnMobSkillCheck(target,mob,skill)
    if (VanadielHour() >= 7 and VanadielHour() <= 18) then
        -- printf("Good to go Moon2");
        return 0;
    else
        return 1;
    end
end;
```

File: scripts/globals/mobskills/Call_of_the_moon1.lua (hunk 1 above)
Comment (xdemolish) [inline scripts/globals/mobskills/Call_of_the_moon1.lua]: dont forget to replace tabs with 4 spaces

File: scripts/globals/mobskills/Call_of_the_moon2.lua (hunk 2 above)
Comment (TeoTwawki) [inline scripts/globals/mobskills/Call_of_the_moon2.lua]: Got same indention weirdness in a couple files.  Just so ya know.

**Comment by teschnei** [comment]
why are there 2 different call of the moons

**Comment by teschnei** [comment]
what is the point of adding a script that just returns 0? that just makes it harder to see which scripts are actually not implemented

File: scripts/globals/mobskills/Dread_Wind.lua [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+-- OnMobWeaponSkill
+---------------------------------------------
+
+function OnMobWeaponSkill(target, mob, skill)
+	local dmgmod = 1;
+
+	local info = MobMagicalMove(mob,target,skill,mob:getWeaponDmg()*3,ELE_WIND,dmgmod,TP_NO_EFFECT);
+	local dmg = MobFinalAdjustments(info.dmg,mob,skill,target,MOBSKILL_MAGICAL,MOBPARAM_WIND,MOBPARAM_WIPE_SHADOWS);
+	
+		if (mob:actionQueueAbility() == true) then
```
Comment (teschnei) [inline scripts/globals/mobskills/Dread_Wind.lua]: none of the actionQueueAbility or ExtraVar stuff is probably going to be used for this.
if it is, set/getExtraVar don't exist anymore and should probably be fixed to use get/setLocalVar

File: scripts/globals/mobskills/Exuviation.lua [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+require("scripts/globals/status");
+require("scripts/globals/monstertpmoves");
+---------------------------------------------
+function OnMobSkillCheck(target,mob,skill)
+	return 0;
+end;
+
+function OnMobWeaponSkill(target, mob, skill)
+
+	local potency = skill:getParam();
```
Comment (teschnei) [inline scripts/globals/mobskills/Exuviation.lua]: script is completely wrong from how the mob version works on retail

File: scripts/globals/mobskills/Jettatura.lua [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+
+function onMobWeaponSkill(target, mob, skill)
+
+	--[[local typeEffect = EFFECT_TERROR;
+	local duration = 2;
+
+	MobGazeMove(mob, target, typeEffect, 1, 0, duration);
+	
+	return typeEffect;]] -- wasn't working correctly
```
Comment (teschnei) [inline scripts/globals/mobskills/Jettatura.lua]: if it's wrong, get rid of it

**Pull request closed without merging**